### PR TITLE
server: Prettier Formatting

### PR DESCRIPTION
**Pull request by znewton**
## Description

Run pnpm run format in Gitrest, Historian, and Routerlicious codebases

Prompted by auto-formatting headache in #14401

### CODE REVIEW

**Review by zhangxin511 — APPROVED** [review]

**Comment by tylerbutler** [comment]
@znewton Thanks for doing this! Based on the lint failure, you may need to adjust the eslint config to extend prettier, or update to the 2.0.0 version of the fluid shared lint config.

**Comment by znewton** [comment]
@tylerbutler thanks for the hint! There was 1 (of 50+) package in R11s with a missing "prettier" extension 😆

**Review by jikim-msft — APPROVED** [review]
Besides the comment that @tylerbutler left above, everything else looks good! Thank you again for working on this!

**Comment by tylerbutler** [comment]
You need to update the root package.json for each release group and add the prettier/prettier:fix scripts to the lint scripts. otherwise there's no CI protection.

File: server/historian/packages/historian-base/src/utils.ts [hunk 1]
```diff
@@ -16,80 +16,80 @@
...
-    return token;
+	let token: string;
+	if (authorization) {
+		const base64TokenMatch = authorization.match(/Basic (.+)/);
+		if (!base64TokenMatch) {
+			throw new NetworkError(403, "Malformed authorization token");
+		}
+		const encoded = Buffer.from(base64TokenMatch[1], "base64").toString();
+
+		const tokenMatch = encoded.match(/(.+):(.+)/);
```
[the remark this answers is not in the document]
Comment (znewton) [reply server/historian/packages/historian-base/src/utils.ts:75]: This was not introduced in this PR

**Comment by znewton** [comment]
@tylerbutler I added the prettier, prettier:fix, prettier:root, and prettier:root:fix scripts to gitrest/package.json, historian/package.json, and r11s/package.json. I also added gitrest/.prettierignore, historian/.prettierignore, and r11s/.prettierignore so that internal package ignore paths were pointing at an actual file. I couldn't reuse the FF/.prettierignore because I wanted to make sure each Docker build pulled in the appropriate prettierignore

**Pull request merged**